static/src/js/components/GranuleDetails/__tests__/test_data.js

```diff
@@ -0,0 +1,189 @@
+}
+
+let formattedGranuleString = '    GranuleUR: 1860_1993_2050_NITROGEN.N-deposition1860.tfw\n'
```

Comment:
`/static/src/js/util/request/__tests__/mocks.js` uses back-ticks and is able to do multiline strings without complaints from the linter -- is this method of concatenating necessary? Also, is the tabbing necessary? This is a brutal variable to maintain.

Reply:
So, this was an interesting one. I had done in order to get around the way vscode was trimming trailing whitespace on the items with `: \n` sequences. I was able to modify the the regex to remove that extra space to `:\n` so this is no longer necessary. As for the tabs, I was following the convention currently used on production. I made the assumption that they were there for a reason.